composer.json

```
@@ -7,7 +7,7 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": ">=7.0",
-        "google/auth": "^1.18.0",
+        "google/auth": "dev-add-get-fetcher as 1.25.0",
```

once `google/auth` is tagged, this will need to be updated to 
```suggestion
        "google/auth": "1.19.1||^1.25.0",
```